### PR TITLE
fix(cli): use slash for alpha values in colors

### LINKED ISSUE #678: bug: CSS classnames opacity is not valid tailwind formatting
When components.json has cssVariables set to false, and you add components, components class names will end up with ":{opacity}" not "/{opacity}" which is incorrect tailwind formatting.

### PR DESCRIPTION
**Pull request by dan5py**
Fixes #678

### CODE REVIEW

**Review by shadcn — APPROVED** [review]
oh nice catch. My bad! I'll review and merge.

**Comment by dan5py** [comment]
@shadcn by bad, forgot the test file

File: packages/cli/test/utils/apply-color-mapping.test.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ describe("apply color mapping", async () => {
       input:
         "text-destructive border-destructive/50 dark:border-destructive [&>svg]:text-destructive text-destructive",
       output:
-        "text-red-500 border-red-500:50 dark:border-red-500 [&>svg]:text-red-500 text-red-500 dark:text-red-900 dark:border-red-900:50 dark:dark:border-red-900 dark:[&>svg]:text-red-900 dark:text-red-900",
```
Comment (shadcn) [inline packages/cli/test/utils/apply-color-mapping.test.ts:67]: oh man! I even got this wrong in the test file 😅

Comment (dan5py) [reply packages/cli/test/utils/apply-color-mapping.test.ts:67]: Happens 😄

**Pull request merged**